### PR TITLE
internal: report cpu time as histogram diff of last time

### PR DESCRIPTION
**Pull request by brettlangdon**
We currently report CPU time as an ever increasing number, instead we want to report it as the difference between the previous CPU time and now in seconds.

### CODE REVIEW

File: ddtrace/internal/writer.py [hunk 1]
```diff
@@ -39,7 +38,9 @@ def __init__(self, hostname='localhost', port=8126, uds_path=None, https=False,
         self.dogstatsd = dogstatsd
         self.api = api.API(hostname, port, uds_path=uds_path, https=https,
                            priority_sampling=priority_sampler is not None)
-        self._stats_rate_counter = 0
+        self._last_thread_time = 0
```
Comment (jd) [inline ddtrace/internal/writer.py]: sounds not necessary

Comment (brettlangdon) [reply ddtrace/internal/writer.py]: you mean, don't initialize if `time.thread_time()` is not available?

Comment (jd) [reply ddtrace/internal/writer.py]: Yes

Comment (brettlangdon) [reply ddtrace/internal/writer.py]:
```suggestion
```

**Pull request merged**